docs/stability.rst

```diff
@@ -257,10 +257,10 @@ The current planned and existing sub-packages are:
                 astropy.units
             </td>
             <td align='center'>
-                <span class="stable"></span>
+                <span class="mature"></span>
             </td>
             <td>
-                New in v0.2. Current functionality stable with intent to maintain backwards compatibility. Significant new functionality, in particular to allow dealing with uncertainties, is likely to be added in future versions.
```

Comment:
So I guess the "uncertainties" thingy is not happening?

Reply:
It is - but it is its own submodule `uncertainty`, which is correctly marked as "in heavy development"